src/main/java/org/toradocu/extractor/JavadocExtractor.java

```diff
@@ -61,7 +61,9 @@ public DocumentedType extract(String className, String sourcePath)
     List<String> classesInPackage = new ArrayList<String>();
     for (File file : listOfFiles) {
       String name = parseClassName(file.getName(), className);
-      if (!name.equals(className) && !name.contains("package-info")) classesInPackage.add(name);
+      if (name != null && !name.equals(className) && !name.contains("package-info")) {
```

Comment:
Why do we exclude classes named `package-info`? We need to document that in the Javadoc.